docs/sdk-and-tools/notifier.md

```diff
@@ -167,11 +168,16 @@ delays due to outport driver.
 
 [comment]: # (mx-context-auto)
 
-### Notifier Service
+## Set up notifier service
 
 In the notifier configuration directory (`cmd/notifier/config`), there is the `config.toml`
 file that can be used to configure the service.
 
+:::tip
+Please make sure to check also [README](https://github.com/multiversx/mx-chain-notifier-go?tab=readme-ov-file#prerequisites)
```

Comment:
_For setting up an events notifier service, make sure to also check the README instructions within the events notifier source code repository._